src/runtime/data/kata-collect-data.sh

```diff
@@ -854,6 +832,18 @@ read_osbuilder_file()
 	cat "$file"
 }
 
+show_latest_kata_version()
+{
+	# kata-check will not run network checks as root user, but this script requires root to run
+	local rc=0
+	local latest_version=$(sudo -u nobody kata-runtime kata-check --check-version-only 2>/dev/null || rc=$?)
```

Comment:
This function should probably honour the `KATA_CHECK_NO_NETWORK` env var. If set, this script should not try to connect to the network.